### DISCUSSION TITLE
怎样让bot正常更新位置

### DISCUSSION THREAD

**Discussion by 223225zzzkkk** [Tnze/go-mc#245] [Q&A] [answered]
怎么样让bot能够被水流、重力等正常改变位置，是不是需要实现java源码中的方法？（顺带一提，当我使用serverbound set player position协议玩家位置没有改变）

<img width="291" alt="2023-05-14 023342" src="https://github.com/Tnze/go-mc/assets/96153949/f028dfc5-3de5-4578-a39d-fcac0c110a55">

求解答

**Comment by patyhank** [comment, marked as answer]
https://github.com/Edouard127/go-mc/
這個人的版本有寫一些 你可以參考看看
大概是在
https://github.com/Edouard127/go-mc/blob/master/bot/provider/basic.go#L358
